index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap" rel="stylesheet">
+    <title>Profolio</title>
+</head>
+
+<body>
+    <header>
+        <h1>Jiachun </h1>
+        <img src="./img/bar-icon.png" />
+    </header>
+
+    <section id="intro">
+        <h1>
+            Hey There. <br />
+            I'm Jiachun
+        </h1>
+        <h2>I'm a Software Developer</h2>
+        <p>
+            I can help you build a product , feature or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded,
+            don’t hestiate to contact me.
+        </p>
```

Comment:
- [ ] In your headline section, kindly ensure that the section has the margins and paddings (top, bottom, left, right) defined in the design. 
- From the images below, you can see that the `<h1>` element isn't properly styled as in the Figma design. 
- It is missing some spacing at the top of the element and secondly, it should be closer to the edge of the element but currently, it looks a little farther from the edge and close to the middle :+1: 

- [ ] Secondly, your `h1` and `h2` text should be properly aligned in the same line as in the Figma design :100: There should also be equal spacing between the elements as seen in the Figma design :+1: 
- Your Design
![image](https://user-images.githubusercontent.com/102699958/226680733-3cfd592f-ea7a-435e-a9df-977462306279.png)

- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/226674898-27806ecf-7bdd-4908-bf44-0755c2a3547a.png)